### DISCUSSION TITLE
issue with using windowed mode

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#2431] [General]
Hey,
So I want to use the windowed mode when playing the game, but because I got multiple monitors it won't let me use it.
What happens is when I use the windowed mode the screen where I got the client on turns black and it doesnt go away. I've seen a couple videos online where I need to input a code in a file in the game folder, but the problem is, I am missing the config.WTF file. Can someone help me with this problem?

**Comment by 55Honey** [comment]
Hi slecht,
Consider joining our discord for even better community interaction.
Here is from our discord's FAQ:

```═════════════════════▣◎▣═══════════════════════

CCpin 4) I try to change my resolution / run the game in Windowed mode - Maximized and all I get is a black screen. 

What should I do? 

Fear not adventurer, there is a task that needs to be done before you engage with the thrilling world of ChromieCraft. Exit the game completely, open the file Config.wtf located in ChromieCraft_3.3.5a/WTF/ and add the following line:

SET gxWindow "1"

You can also set your, FHD for example, resolution by changing or adding if it's not there, the line:

SET gxResolution "1920x1080"```

**Comment by ayanimea** [comment]
Hello,
I ran into the same issue and the SET gxWindow did correct the issue but changing the setting ended up in a much bigger latency. Do you have any idea about what may be causing this?
Reverting the settings by going back to fullscreen mode now causes the same black screen.
Cheers.

**Comment by TheSCREWEDSoftware** [comment]
Windowed modes uses more resources, should it impact your connection? Only if you're playing at very low frames. 

Keep in mind that we're few crashes during the past week

**Reply by ayanimea** [reply]
> Windowed modes uses more resources, should it impact your connection? Only if you're playing at very low frames.

I play at maximum resolution and, before using windowed mode, I had absolutely no issue. About my computer, I don't have a GPU, only Intel chipset, but 32 GB of RAM.
It may be an issue with a conflict between DirectX versions, I heard. Do you confirm that ChromieCraft uses DX9? Because, if so, maybe I need to tweak my configuration a bit.

> Keep in mind that we're few crashes during the past week

I didn't know about the crashes, I'm new to ChromieCraft. So, I'll check if my connection issues get better in a few days. But, after deleting `Config.wtf`, I had no issue with the connection or latency anymore, so I'm not sure if it's on the server's side.

If I resume the tests and their results:
- Without any modification of `Config.wtf`, I can't use the windowed mode: the screen is totally black, even after rebooting the game
- If I select the windowed mode and update manually `Config.wtf `by adding `SET gxWindow "1"`, the windowed mode works, but I have a really high latency and frequent disconnections.
- If I select, from the game's UI, the fullscreen mode again, the screen is black, even after rebooting the game.
- If I delete `Config.wtf` and restart the game, I'm back to the former, excellent performances and I can't use the windowed mode anymore (black screen).

My hunch is that it's on my side, either the client or DirectX. And, since the former user had that issue because of a dual-screen, I'll guess my own dual-screen may be a factor.
